vulnerablecode/settings.py

```diff
@@ -160,5 +160,5 @@
         "rest_framework.filters.SearchFilter",
     ),
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
-    "PAGE_SIZE": 100,
+    "PAGE_SIZE": 10,
```

Comment:
Isn't this enough? The paginate_by on ViewSet is only useful to override the default setting, no?

Reply:
@tdruez yes, you are right, this is enough